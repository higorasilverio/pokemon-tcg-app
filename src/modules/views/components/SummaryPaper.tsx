import { ReactNode } from 'react'
import styles from '../../../styles/SummaryPaper.module.scss'

const SummaryPaper = ({ children }: { children: ReactNode }) => {
  return <div className={styles.paper}>{children}</div>
}

export default SummaryPaper
