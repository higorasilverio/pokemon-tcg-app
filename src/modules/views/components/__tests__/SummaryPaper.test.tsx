import '@testing-library/jest-dom'
import { render, screen } from '@testing-library/react'
import SummaryPaper from '../SummaryPaper'

describe('SummaryPaper', () => {
  it('renders unchanged', () => {
    const { container } = render(<SummaryPaper>Summary</SummaryPaper>)

    expect(container).toMatchSnapshot()
  })

  it('should render correctly', () => {
    render(<SummaryPaper>Summary</SummaryPaper>)

    const text = screen.getByText(/summary/i)

    expect(text).toBeInTheDocument()
  })
})
