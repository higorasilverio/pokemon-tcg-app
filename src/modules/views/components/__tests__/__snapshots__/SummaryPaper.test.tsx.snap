// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SummaryPaper renders unchanged 1`] = `
<div>
  <div
    class="paper"
  >
    Summary
  </div>
</div>
`;
